blog/source/_posts/MIDI-velocity-transformer.md

```diff
@@ -0,0 +1,266 @@
+---
+title: MIDI Velocity Prediction with Transformer
+date: 2023-09-11
+tags:
+author: wmatejuk
+---
+
+### Why is it important?
+
+{% algrtmImgBanner MIDI-velocity-transformer/banner-1.jpg pianoroll %}
+
+MIDI velocity is a crucial element in music dynamics, determining the force with which a note is played, 
+which profoundly influences the emotional quality of music. 
+
+If you were to take a sequence of notes and predict their velocities by an untrained model, this is what you wold have ended up with:
```

Comment:
Those samples could be a good demonstration of what velocity is and how it affects the music - you could say that the velocities were randomized here.

Saying it's generated by an untrained model is confusing here, because you haven't introduce the problem that the model has to solve yet.